src/extensions/extension.tsx

```diff
@@ -35,10 +35,10 @@ import {
   // https://github.com/satya164/react-simple-code-editor/issues/86
   const reactSimpleCodeEditorFix = document.createElement("script");
   reactSimpleCodeEditorFix.innerHTML = "const global = globalThis;";
-  document.head.appendChild(reactSimpleCodeEditorFix);
+  document.body.appendChild(reactSimpleCodeEditorFix);
 
   // Show message on start.
-  console.log(`Initializing Spicetify Marketplace v${MARKETPLACE_VERSION}`);
+  // console.log(`Initializing Spicetify Marketplace v${MARKETPLACE_VERSION}`);
```

Comment:
Can we leave this one in? We added it so it's easy to see if someone has marketplace installed properly.

Reply:
its your wish. I just commented it so dev tools wont look clumsy